docs/how-to/download-new-upstream-version.rst

```diff
@@ -7,3 +7,63 @@ Download a new Upstream Version
 .. caution::
 
     Work in progress!
+
+Once in a while you need to download a new upstream release or check if a newer 
+upstream release exists.
+
+Most of the source packages contain a ``watch`` file in the ``debian`` folder.
+This is a configuration file for the :manpage:`uscan(1)` utility which allows
+you to automatically scan ``http`` or ``ftp`` sites or :manpage:`git(1)`
+repositories for newly available updates of the upstream project.
+
+.. note::
+    If the source package does not contain a ``watch`` file, there may be
+    an explaination and instructions in the ``README.source`` or 
+    ``README.debian`` file (if available) that tell you how to proceed.
+
+.. caution::
+    Only if there is no automatic download mechanism and you can't find any
+    download instructions, you should download any upstream file manually.
+
+    Remember that a package may get distributed to hundred thousands of users.
+    Humans are the weakest link in this distribution chain, because we may
+    accidentally miss or skip a verification step, misspell a URL, copy the 
+    wrong URL or copy a URL only partially, etc.
+
+If you just want to check if a new update is available, but you don't want to 
+download anything you can run the following command from the root of the 
+source tree::
+
+    uscan --safe
+
+.. note::
+    If :manpage:`uscan(1)` could not find a newer upstream version it will 
+    return with the exit code `1` and print nothing to the standard output.
+
+You can add the ``--verbose`` flag to see more information (e.g., which version 
+:manpage:`uscan(1)` found)::
+
+    uscan --safe --verbose 
+
```

Comment:
Might be worth adding a subheading on this line here, it seems sort of separate enough from the previous content.